components/Code.tsx

```diff
@@ -1,7 +1,18 @@
-export default function Code({ children }) {
+import { cn } from "@/lib/utils";
+
+export default function Code({
+  children,
+  noWrap,
+}: {
+  children: React.ReactNode;
+  noWrap?: boolean;
+}) {
   return (
     <code
-      className={`whitespace-nowrap rounded-md bg-tan-300 px-1.5 py-1 text-sm text-stone-600`}
+      className={cn(
+        `rounded-md bg-tan-300 px-1.5 py-1 text-sm text-stone-600`,
+        noWrap && `whitespace-nowrap`,
```

Comment:
If it's an override of the default appearance, I'd prefer we add a `className` prop that's passed as the final child in this `cn("...", className)`. Then you'd call `<Code className="whitespace-normal" />` if thatʻs what youʻre trying to achieve.

I'm reeeeally trying to get away from mucking up the API of our components, avoiding  one-off props that are for a single class or style override.

Reply:
Also... wait Iʻm confused. The only reference to `<Code />` in this PR uses `noWrap`? Whatʻs this change achieve?

Reply:
I don't think I changed this at all, should I revert these changes?

<img width="1359" alt="Screen Shot 2024-05-07 at 4 27 59 PM" src="https://github.com/hawaiians/hawaiiansintech/assets/26368729/7efb20df-dc60-4d8a-9c3a-4214ee6f582a">

Reply:
ok for some reason without this change the deployment breaks - https://vercel.com/hawaiians-projects/hawaiiansintech/DzUjkWVgXsf18PTq3Hng6bEQ35ef

 so I'm gonna put these changes back for now, let's come back to this in the future if we need